AsyncDisplayKit/Layout/ASStaticLayoutSpec.h

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <AsyncDisplayKit/ASLayoutSpec.h>
+#import <AsyncDisplayKit/ASStaticLayoutSpecDimension.h>
+
+@interface ASStaticLayoutSpecChild : NSObject
```

Comment:
There is a [follow-up issue](https://github.com/facebook/AsyncDisplayKit/issues/539) for this. I will probably make some sample usages of this spec.